disnake/audit_logs.py

```diff
@@ -758,8 +758,8 @@ def _convert_target_guild(self, target_id: int) -> Guild:
     def _convert_target_channel(self, target_id: int) -> Union[abc.GuildChannel, Object]:
         return self.guild.get_channel(target_id) or Object(id=target_id)
 
-    def _convert_target_user(self, target_id: int) -> Union[Member, User, None]:
-        return self._get_member(target_id)
+    def _convert_target_user(self, target_id: int) -> Union[Member, User, Object, None]:
+        return self._get_member(target_id) or Object(id=target_id) if target_id else None
```

Comment:
Instead of checking `if target_id` here, it would probably make sense to move that check into `AuditLogEntry.target`.
The `getattr` there currently also hides the fact that it's passing `Optional[int]` to `converter`, which only accepts `int`.

Reply:
👍

Reply:
changed it [here](9970b3a164820f812b1abe9a470f3081c40093cb), but I'll keep this conversation open since we should document that `.target` might now return `None` for target types that aren't expected to get `None`

Reply:
> document that `.target` might now return `None` for target types that aren't expected to get `None`

Does it? It'll return `None` if `target_id = None`, but if it's a non-null value it falls back to `Object`, as far as I can tell.

Reply:
Yeah you're right, I'll mark this as resolved 😄